svg/svg.go

```diff
@@ -11,28 +11,28 @@ package svg
 func GetBullet(color string) string {
 	switch color {
 	case "":
-		return `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="none" width="16" height="16"><defs><path fill="#fff" fill-opacity="0" d="M16 0H0v16h16V0z" id="a"/></defs><use xlink:href="#a"/></svg>`
+		return `<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"></svg>`
 	default:
-		return `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="none" width="16" height="16"><defs><path fill="` + color + `" d="M5.85 13.1q.9.9 2.15.9 1.3 0 2.1-.9.9-.85.9-2.1t-.9-2.15Q9.3 8 8 8q-1.25 0-2.15.85Q5 9.75 5 11t.85 2.1z" id="a"/></defs><use xlink:href="#a"/></svg>`
```

Comment:
Most of the changes are just unwrapping the `defs` node & removing the ID of the path nested in it.